server/handlers/tripHandler.js

```diff
@@ -24,12 +24,14 @@ let getTrips = async (req, res) => {
  * @returns {Promise<void>} the promise indicating success
  */
 let addTrip = async (req, res) => {
+  console.log(req.body);
   const query = `INSERT INTO trips (user_id, name, dscript)
                   VALUES (
-                    (SELECT user_id FROM users
-                    WHERE email = ?), ?, ?
-                  )`;
-  const params = [req.body.email, req.body.name, req.body.dscript];
+                    (SELECT MAX(user_id) 
+                     FROM users
+                     WHERE email = ?), ?, ?
+                    )`;
+  const params = ["Billybob", "React trip", "Bogus"]; //[req.body.email, req.body.name, req.body.dscript];
```

Comment:
This fix makes sense; it might even be applicable to all handlers (if it's not already done). I think it's indicative of just how severe our issue with duplicate entries is (but that's an issue that's still on the table in our sprint).

Reply:
Agreed.